CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
+## v1.6.6
+
+- Fixed the drag over colors
```

Comment:
Can you elaborate a bit more? I don't know what this means.

Reply:
Spruced that up a bit.